helpers/config.js

```diff
@@ -164,6 +164,15 @@ function Config(packageJson, parseCommandLineOptions = true) {
 
 		validateForce(appConfig);
 
+		// Disable api, peers, broadcasts, syncing and ws workers when snapshotting mode required
+		if (program.snapshot) {
+			appConfig.api.enabled = false;
+			appConfig.peers.enabled = false;
+			appConfig.peers.list = [];
+			appConfig.broadcasts.active = false;
+			appConfig.syncing.active = false;
+			appConfig.wsWorkers = 0;
```

Comment:
I really don't like setting wsWokers here. Or overriding the peers list. 

Also please put some logging here for user that you are overriding or disabling these values for snapshotting.

Reply:
All these parameters affect the snapshot mode. They were present but not used till now.

Regarding the logging suggestion, the user requests a snapshot and it's been notified via https://github.com/LiskHQ/lisk/blob/d4736c1f05a8c87e2e25d6cb273c6ac07361e4f5/modules/loader.js#L766 .  i don't think we need to add anything else than a code comment.